docs/physics/montecarlo/initialization.ipynb

```diff
@@ -71,18 +71,54 @@
   },
```

Comment:
Hi @smithis7 

Thanks for taking the time to review my code and for your suggestion!

I followed your suggestion, and got interesting behavior. You can see it [here](https://github.com/smokestacklightnin/tardis/blob/no-force-compile/docs/physics/montecarlo/initialization.ipynb). I haven't added it to my pull request because I want to investigate the behavior for the coarsest grid where all the implementations perform basically the same. This is obviously the result of the JIT compilation, but it doesn't feel right it should take *so much* time (unless my intuition is skewed). I would like to isolate the behavior as much as possible.

Do you or any of the other members have time for a video or audio call with me? I would love some feedback on the draft of my application and to discuss my idea for this project in more depth.

I am in Southern California and afternoons usually work for me, but I can be flexible to accommodate your schedule.

I will reach out on Gitter as well, so we can discuss the details there further.